uberfire-simple-docks/uberfire-simple-docks-client/src/main/java/org/uberfire/client/docks/UberfireDocksImpl.java

```diff
@@ -202,4 +206,17 @@ private boolean dockIsEnable( UberfireDockPosition dockPosition ) {
         }
         return true;
     }
+
+    private void refreshDocksSizeOnWindowResize() {
+        Window.addResizeHandler( new ResizeHandler() {
```

Comment:
@paulovmr  please try to add this handler on the composite and not in the window.